ui/components/app/confirm-page-container/confirm-page-container.component.js

```diff
@@ -230,7 +230,7 @@ const ConfirmPageContainer = (props) => {
             <ActionableMessage
               message={
                 isBuyableChain ? (
-                  <Typography variant={TypographyVariant.H7} align="left">
+                  <Text variant={TextVariant.bodySm} align="left">
```

Comment:
Some props are a little different on the Text component
```suggestion
                  <Text variant={TextVariant.bodySm} textAlign="left">
```